Feat: Schedule Entity 작성

LEESEUNGRYEOL opened the pull request:
## 📕 제목
- #13, #14

## 📗 작업 내용

> 구현 내용 및 작업 했던 내역

- [x] 작업내용1 :  Schedule Entity 작성 (CRUD)
- [x] 작업내용2 : GPT를 이용한 자동 PR 생성


## 📘 PR 특이 사항

> PR을 볼 때 주의깊게 봐야하거나 말하고 싶은 점

- 특이사항1

LEESEUNGRYEOL committed:
Feat: Schedule Service 구현 #13

1. Schedule Service

LEESEUNGRYEOL committed:
Chore: OPENAI 를 이용한 PR 자동작성 #14

1. OPENAI 를 이용한 PR 자동작성을 위한 .yml 파일 생성

reeruryu commented on .github/workflows/codeReview.yml:
```diff
@@ -0,0 +1,21 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          LANGUAGE: Korean
\ No newline at end of file
```
openai 짱이네 진짜

reeruryu commented on src/main/java/com/seungah/todayclothes/controller/ScheduleController.java:
```diff
@@ -0,0 +1,42 @@
+@RestController
+@RequiredArgsConstructor
+public class ScheduleController {
+
```
@RequestMapping("/api/schedule")로 시작하면 좋을 것 같아요

reeruryu commented on src/main/java/com/seungah/todayclothes/dto/request/ScheduleRequest.java:
```diff
@@ -0,0 +1,12 @@
+    private String title;
+    private LocalDate date;
+    private String location;
+}
```
@github-actions 처럼 validation 체크하면 좋을 듯 싶어요~!

reeruryu commented on src/main/java/com/seungah/todayclothes/repository/ScheduleRepository.java:
```diff
@@ -4,7 +4,10 @@
 public interface ScheduleRepository extends JpaRepository<Schedule, Long> {
 
+    List<Schedule> findByMemberId(Long userId);
 }
```
진짜 좋네.. memberId로 변수명 바꾸면 좋을 것 같아요 ㅎㅎㅋㅋㅋㅋ

reeruryu commented on src/main/java/com/seungah/todayclothes/service/ScheduleService.java:
```diff
@@ -0,0 +1,73 @@
+        );
+
+        scheduleRepository.deleteById(id);
+        return ResponseEntity.ok().build();
```
findById, deleteById -> 조회가 2번 일어나는 것 같아요. delete로 바꾸면 좋겠네용

reeruryu reviewed (commented):
고생하셨우요 ㅎㅎ

dmdwn99 committed:
Merge branch 'develop' into feat/Schedule

# Conflicts:
#	src/main/java/com/seungah/todayclothes/common/exception/ErrorCode.java

dmdwn99 commented:
확인했습니다!

dmdwn99 committed:
Refact: Schedule Refactoring #13

1. Schedule 저장 시 Member 누락 추가
2. Controller @RequestMapping("/api/schedule") 일괄 처리
3. Request Validation 추가
4. Service deleteById를 delete로 수정
